UACloudLibraryGraphQLClient/UACloudLibClientLibrary/Models/InfoModell.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UACloudLibClientLibrary
+{
+    public enum AddressSpaceLicense
+    {
+        MIT,
+        ApacheLicense20,
+        Custom
+    }
+    
+    /// <summary>
+    /// Contains the metadata of the nodeset and the nodeset itself
+    /// </summary>
+    public class AddressSpace
+    {
+        public AddressSpace()
+        {
+            this.Nodeset = new AddressSpaceNodeset2();
+            this.Contributor = new Organisation();
+            this.Category = new AddressSpaceCategory();
+        }
+        public string ID {get; set; }
```

Comment:
This file has changed, see https://github.com/OPCFoundation/UA-CloudLibrary/blob/main/UACloudLibraryServer/Models/InfoModel.cs. For example, IDs are gone.